### ISSUE TITLE
Composicao no coinr é apenas nivel para nivel + 1

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [AdaptaBrasil/reprodutibilidade#7]
Verificar se o coinr exige que a composicao seja de nivel n para nivel n + 1. Se for, teremos que contornar isto para trabalhar com os dados do adapta.

**Comment by netebarreto** [comment]
Sim, o Coinr utiliza do nivel _n_ para o _n+1_,  mas não é complicado de contornar, basta fazer _[ n - (max(n)-1)]_  para ajustar o adapta ao Coinr.

**Comment by pedro-andrade-inpe** [comment]
Sim, usar o complemento faz parte da solução. Mas a questao aqui é que o Adapta pula niveis. Veja por exemplo uma das tabelas do setor de saude:

![image](https://github.com/user-attachments/assets/806650d9-22a9-4d4e-9693-3e1a74994500)

Acredito que o coinR nao acomode aqueles buracos nos niveis 4 e 5. Provavelmente teremos que fazer composicoes do tipo inidicador do nivel x é igual ao indicador do nivel x - 1 para acomodar esses buracos.

**Comment by pedro-andrade-inpe** [comment]
Os niveis do adapta sao bem amarrados. O nivel 4 só tem sensibilidade e capacidade adaptativa, entao no indice de exposicao tambem tem o mesmo problema. No caso da exposicao pula um nivel, no caso da ameaca pula dois niveis. Acho que este padrao se aplica a todos ou quase todos os setores do adapta. Mas é importante falar que o software nao tem esta restricao, entao precisamos nos preparar para buracos em qualquer lugar nesta tabela. As tabelas dos setores atualmente publicados estao em 

https://adaptabrasil.mcti.gov.br/index.php/sobre/lista-de-indicadores
